samples/accounts_user_links_audit_test.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import accounts_user_links_audit
+
+
+TEST_ACCOUNT_ID = os.getenv("GA_TEST_ACCOUNT_ID")
+
+
+def test_accounts_user_links_audit(capsys):
+    accounts_user_links_audit.audit_account_user_links(TEST_ACCOUNT_ID)
+    out, _ = capsys.readouterr()
+    assert "Result" in out
```

Comment:
It would probably make sense to add a few more asserts for your other print statements just to be super sure they happen